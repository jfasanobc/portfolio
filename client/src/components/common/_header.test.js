import React from 'react'
import renderer, { create } from "react-test-renderer";
import HeaderBlock from './header.js'

describe("Render HeaderBlock", () => {
  test("Is it rendered and matching", () => {
    const button = create(<HeaderBlock />)
    expect(button.toJSON()).toMatchSnapshot()
  })
})