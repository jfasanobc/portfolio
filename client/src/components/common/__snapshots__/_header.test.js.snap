// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render HeaderBlock Is it rendered and matching 1`] = `
<div
  className="grid"
>
  <div
    className="expanding_nav"
  >
    <img
      alt="Open Menu"
      className="menu_icon center"
      src="./assets/img/menu.png"
    />
    <ul
      className="nav"
    >
      <li
        className="menu_item"
        title="Home"
      >
        <a
          alt="Home"
          href="#"
        >
          Home
        </a>
      </li>
      <li
        className="menu_item"
        title="About Me"
      >
        <a
          alt="About Me"
          href="#about-me"
        >
          About Me
        </a>
      </li>
      <li
        className="menu_item"
        title="Contact Me"
      >
        <a
          alt="Contact Me"
          href="#contact-me"
        >
          Contact Me
        </a>
      </li>
    </ul>
  </div>
</div>
`;
